health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/MapDrillChart.js

```
@@ -0,0 +1,135 @@
+    >
+        <div>
+          {" "}
+          <div style={{ float: "left" }}>
+            <Backsvg onClick={onBack} />
+          </div>
+          {data2 && data2.length == 0 && (
+            <div style={{ paddingTop: "60px" }}>
+              {t("DSS_NO_DATA")}
+            </div>
+          )}
+          {data2 && data2[0] && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)